### ISSUE TITLE
Saved model is not the best model

### ISSUE THREAD

**Issue by davidecarnevali** [eugeniaring/Pytorch-tutorial#1]
Hi,
in your code you save the model at the end of the k-fold cross validation for loop, but in this way you just save the model with the state_dict of the last epoch of the last fold, which is not assured it is the "best model"...
Shouldn't be better to save it only when the val accuracy reached the highest value?
Thx

D.
